sendsoknad-business/src/main/java/no/nav/sbl/dialogarena/soknadinnsending/business/db/soknad/SoknadRepositoryJdbc.java

```diff
@@ -484,6 +484,15 @@ public List<Faktum> hentAlleBrukerData(String behandlingsId) {
         return fakta;
     }
 
+    public void slettGamleSoknader() {
+        int eightWeeks = 8 * 7;
+        String sql = "select soknad_id from soknad where sistlagret < CURRENT_TIMESTAMP - (INTERVAL '" + eightWeeks + "' DAY)";
+        List<Long> ids = getJdbcTemplate().queryForList(sql, Long.class);
+
+        logger.info("Fant {} soknader eldre enn {} dager. Sletter soknader med disse ids: {}", ids.size(), eightWeeks, ids);
+        ids.forEach(this::slettSoknad);
```

Comment:
Det er relativ høy latensitet mellm Oracle og GCP. Kan du slette alle søknader på en gang ?

Reply:
Det kan jeg absolutt, men trengs det?

Igår ble det 232 soknader opprettet, som ikke ble sendt inn eller avsluttet. Så hvis det kommer å vare ca 230 soknader som denne spörringen sletter, og dette jobbet kjöres 04 på natta, er det virkelig et problem?

Jeg tenker at fordelen med at kalle `slettSoknad()` er at vi ikke trenger å oppdatere logikken for å slette soknader på flere sted, hvis vi i framtiden trenger å endre. Men hvis du trur at dette tar for lang tid så kan jeg selvsagt gjöre en egen spörring for at slette allt på en gang.

Reply:
Kanskje vi ikke trenger det. Jeg bare pekker på dette med latensiteten. Husker du vi hadde mange Connection Pool exhausted type feil når vi ikke passet på det.